src/main/java/Duke.java

```diff
@@ -1,10 +1,115 @@
+            throw new StringIndexOutOfBoundsException();
+        }
+        int taskNumber = Integer.parseInt(arrayOfCommand[1]);
+        if (arrayOfCommand[0].equals("mark")) {
+            markTask(tasks, taskNumber - 1);
+        } else {
+            unmarkTask(tasks, taskNumber - 1);
+        }
+    }
+
+    public static void handleTodoDeadlineAndEvent(String userCommand,int taskCount,String[] arrayOfCommand,Task[] tasks){
+        if (userCommand.startsWith("todo")) {
+            arrayOfCommand = userCommand.split(" ", 2);
+            if (arrayOfCommand.length < 2 || arrayOfCommand[1].isEmpty()) {
+                System.out.println(Guide.TODO_REQUEST_FORMAT);
+                throw new StringIndexOutOfBoundsException();
+            }
+            tasks[taskCount] = new Todo(arrayOfCommand[1]);
+        }
+
+        if (userCommand.startsWith("deadline")) {
+            arrayOfCommand = userCommand.split("deadline | /by");
+            if (arrayOfCommand.length < 2 || arrayOfCommand[1].isEmpty()) {
+                System.out.println(Guide.DEADLINE_REQUEST_FORMAT);
+                throw new StringIndexOutOfBoundsException();
+            }
+            tasks[taskCount] = new Deadline(arrayOfCommand[1], arrayOfCommand[2]);
+        }
+
+        if (userCommand.startsWith("event")) {
+            arrayOfCommand = userCommand.split("event | /from | /to ");
+            if (arrayOfCommand.length < 3 || arrayOfCommand[1].isEmpty() || arrayOfCommand[2].isEmpty()) {
+                System.out.println(Guide.EVENT_REQUEST_FORMAT);
+                throw new StringIndexOutOfBoundsException();
+            }
+            tasks[taskCount] = new Event(arrayOfCommand[1], arrayOfCommand[2], arrayOfCommand[3]);
+        }
+
+        System.out.println("Got it. I've added this task:");
+    }
+
+    public static void handleCommand(String userCommand) {
+        Task[] tasks = new Task[100];
+        String[] arrayOfCommand = new String[4];
+        int taskCount = 0;
+        Scanner in = new Scanner(System.in);
+
+        while (!(userCommand.equals("bye"))) {
+            if (userCommand.equals("list")) {
+                printTasks(tasks, taskCount);
+                userCommand = in.nextLine();
+                continue;
+            } else if (userCommand.contains("unmark")) {
+                handleMarkAndUnmarkRequest(userCommand, arrayOfCommand, tasks);
+                userCommand = in.nextLine();
+                continue;
+            } else if (userCommand.contains("mark")) {
+                handleMarkAndUnmarkRequest(userCommand, arrayOfCommand, tasks);
+                userCommand = in.nextLine();
+                continue;
+            } else if (userCommand.contains("todo") || userCommand.contains("deadline") || userCommand.contains("event")) {
+                handleTodoDeadlineAndEvent(userCommand, taskCount, arrayOfCommand, tasks);
+            } else { System.out.println(Guide.REQUESTS_FORMAT);
```

Comment:
The `System.out.println(Guide.REQUESTS_FORMAT);` should on the next line instead of this line